src/core_plugins/region_map/public/region_map_vis.js

```diff
@@ -0,0 +1,81 @@
+import './region_map.less';
+import './region_map_controller';
+import './region_map_vis_params';
+import regionTemplate from './region_map_controller.html';
+import image from './images/icon-vector-map.svg';
+import { TemplateVisTypeProvider } from 'ui/template_vis_type/template_vis_type';
+import { VisSchemasProvider } from 'ui/vis/schemas';
+import { VisTypesRegistryProvider } from 'ui/registry/vis_types';
+import { VisVisTypeProvider } from 'ui/vis/vis_type';
+import { truncatedColorMaps } from 'ui/vislib/components/color/truncated_colormaps';
+
+VisTypesRegistryProvider.register(function RegionMapProvider(Private, vectormapsConfig) {
+
+  const VisType = Private(VisVisTypeProvider);
+  const TemplateVisType = Private(TemplateVisTypeProvider);
+  const Schemas = Private(VisSchemasProvider);
+
+  const vectorLayers = vectormapsConfig.layers.slice();
```

Comment:
This should only be being called once for the "client-side app" when it's registering all the visualization types, what are we trying to prevent with the  `slice()`? I know before that we were having layers duplicated when creating multiple maps in a row, was this meant to address this?